src/librustc_borrowck/borrowck/mod.rs

```diff
@@ -971,13 +978,11 @@ impl<'a, 'tcx> BorrowckCtxt<'a, 'tcx> {
                             let span = self.tcx.map.span(local_id);
                             if let Ok(snippet) = self.tcx.sess.codemap().span_to_snippet(span) {
                                 if snippet != "self" {
-                                    db.span_suggestion(
-                                        span,
-                                        &format!("to make the {} mutable, use `mut` as shown:",
-                                                 self.cmt_to_string(&err.cmt)),
-                                        format!("mut {}", snippet));
+                                    db.span_label(span,
+                                        &format!("use `mut {}` here to make mutable", snippet));
```

Comment:
I think `span_suggestion` should be used so that machines can read them, but it should be rendered like label for humans.